docs/reference/aggregations/pipeline/normalize-aggregation.asciidoc

```diff
@@ -0,0 +1,119 @@
+[[search-aggregations-pipeline-normalize-aggregation]]
```

Comment:
This'll want `[role="xpack"]` or so it'll get the little xpack bug.

Reply:
added role!